hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/NodeInfoTest.java

```diff
@@ -120,7 +120,7 @@ void logsErrorOnMissingAccountForNonZeroStake() {
 
     @Test
     void doesNotLogErrorOnMissingAccountForZeroStake() {
-        givenEntryWithMemoAndStake(nodeId, "Oops!", 0L);
+        givenEntryWithMemoAndWeight(nodeId, "Oops!", 0L);
```

Comment:
```suggestion
        givenEntryWithMemoAndStake(nodeId, "Oops!", 0L);
```

Reply:
This is on the address book. Should we not leave it?

Reply:
I updated all other variables and methods except for these two. It made sense to me to leave them as *Weight because they are calling address.getWeight() to determine what value to use.

Reply:
Yeah makes sense